code/modules/vehicles/motorbike.dm

```diff
@@ -53,6 +54,9 @@
 	return welder_repair_act(user, I, 10, 2 SECONDS, fuel_req = 1)
 
 /obj/vehicle/ridden/motorbike/relaymove(mob/living/user, direction)
+	if(!inserted_key)
+		to_chat(user, span_warning("You need the key to use the bike!"))
+		return
```

Comment:
Shouldn't this be on relay drive or something to fix this?

Reply:
Well, given there's the fuel check just below, I figured it was fine